src/inject/index.js

```diff
@@ -110,7 +110,7 @@ function getNewContent(target, collection, opt) {
       var pluralState = filesCount > 1 ? 's' : '';
       log(cyan(filesCount) + ' file' + pluralState + ' into ' + magenta(target.relative) + '.');
     } else {
-      log('Nothing to inject into ' + magenta(target.relative) + '.');
+      error('Nothing to inject into ' + magenta(target.relative) + '.');
```

Comment:
I also suggest adding a test for this

Reply:
Added tests for both error and log if there is nothing to inject